src/js/common/utils/validations.js

```diff
@@ -242,7 +242,7 @@ function isValidSSN(value) {
     return false;
   }
 
-  return /^\d{3}-?\d{2}-?\d{4}$/.test(value);
+  return /^\d{9}$/.test(value) || /^\d{3}-\d{2}-\d{4}$/.test(value);
```

Comment:
How is this any different? I guess this way, if they have one `-`, they have to have the second, but...was that the problem?

Alternatively, would it be out of scope to strip non-numeric characters and force-format it with dashes?